src/components/GraphFilter/GraphFind.tsx

```diff
@@ -47,21 +51,28 @@ export class GraphFind extends React.PureComponent<GraphFindProps> {
       props.toggleFindHelp();
     }
 
-    this.hideInputRef = React.createRef();
-    this.hideInputValue = '';
-    this.hideValue = '';
-    this.hiddenElements = undefined;
-
     this.findInputRef = React.createRef();
     this.findInputValue = '';
-    this.findValue = '';
+
+    this.hiddenElements = undefined;
+
+    this.hideInputRef = React.createRef();
```

Comment:
I see in the code that you are doing:

```
inputRef={ref => {
                  this.findInputRef = ref;
                }}
```

It doesn't make sense to use `React.createRef()`, this will cause confusion because react ref use `theref.current` and this doesn't.

The same applies for the `findInputRef`

Reply:
OK, I removed the two React.createRef lines and it does seem to work.